packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -174,7 +173,7 @@
             slot="body"
             bind:filters={tempFilters}
             {bindings}
-            {schemaFields}
+            {table}
```

Comment:
Would it be cleaner here to just add the ID and rev fields to the `schemaFields` value that we already work out in this file, and then avoid having to make any changes to the filter drawer?

The function you put inside the filter drawer is exactly the function we would just need to put in here, and then we could keep the interface of filter drawer the same.

It's probably much of a muchness, but I'd lean towards keeping the filter drawer prop interface the same rather than having 2 possible props passed in to determine the schema.

The other option which might actually be even cleaner, would be actually just using `getSchemaForDatasource` inside here. You're right that it's sort of tailored towards the client, but the invocation to use it here would just be:
`$: schemaFields = getSchemaForDatasource(null, { type: "table", tableId })`

The only issue with using `getSchemaForDatasource` is that it adds `_id` and `_rev` for all tables, not just internal. Technically `_id` is in the schema of all tables, but `_rev` should only be internal, so that's a change we should probably make in that file. This use case is different though because this is like the "searchable" schema, which means we don't want to include `_id` for non-internal tables.

I think overall then my preference would be to pull the function from the filter drawer inside here, and just work out the full schema fields inside here, and make no changes to the filter drawer. And then we won't use `getSchemaForDatasource` at all, since we don't want the readable schema.

What are your thoughts? Do you prefer adding it to the filter drawer?